### ISSUE TITLE
Package suggestion: `yup-phone-lite` for `JSON schema validation`

### ISSUE THREAD

**Issue by csandman** [pastelsky/bundlephobia#760]
[`yup-phone-lite`](https://www.npmjs.com/package/yup-phone-lite)

**Alternative to**

[`yup-phone`](https://www.npmjs.com/package/yup-phone)

**Quality check**

- [x] Package has sufficient overlap in functionality to act as a replacement.
- [x] Package is actively maintained, and/or stable for use.
- [x] Package has at least 1000 weekly downloads on NPM or is relatively popular on GitHub.
- [x] This package is a better alternative to what is already suggested for this category (please explain why), or the category is new.

`yup-phone-lite` offers the same functionality as `yup-phone` with a dramatically smaller bundle size.

**Comment by pastelsky** [comment]
yup-phone-lite is a focused Yup extension for phone-number validation, not a general runtime schema-validation library. It therefore does not fit this category as a same-purpose replacement.

**Comment by csandman** [comment]
@pastelsky fair enough, this was posted three years ago haha, I forgot I even opened this issue
